apps/src/codebridge/FileBrowser/hooks/useHandleDragEnd.ts

```diff
@@ -40,11 +42,13 @@ export const useHandleDragEnd = () => {
           return;
         }
         if (e.active.data.current?.type === DragType.FOLDER) {
-          const validationError = validateFolderName({
-            folderName: source.folders[e.active.data.current.id].name,
-            parentId: e.over.id as string,
-            projectFolders: source.folders,
-          });
+          const folderId = e.active.data.current.id as string;
+          const validationError = validateFolderMove(
```

Comment:
Just wondering if the check for whether the folder is being moved into a child folder needed now that `fileBrowserKeyboardCoordinateGetter` and `fileBrowserCollisionDetector` are being used by DND.

Reply:
I _think_ you are right, but I think I might do you other suggestion and use this function to validate folder moves in both the drag & drop and move via dropdown cases, just to be doubly safe.

Reply:
Sounds good!